notebooks/03_global_forecasting.ipynb

```diff
@@ -20,9 +20,7 @@
     "* Foundation Models\n",
```

Comment:
<p>Not only for various tasks (I think various tasks is still a bit of future work and slightly adressed). It is more about that it is designed to perform well on various different types of times series </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-euroscipy2024/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='95'/>